src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -911,7 +911,8 @@ impl_display_t!(KafkaSinkConfigOption);
 
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub enum PgConfigOptionName {
-    /// Hex encoded string of binary serialization of `dataflow_types::PostgresSourceDetails`
+    /// Hex encoded string of binary serialization of
+    /// [`mz_storage_types::sources::postgres::PostgresSourcePublicationDetails`]
```

Comment:
@petrosagg this is erroring on
```
error: unresolved link to `mz_storage_types::sources::postgres::PostgresSourcePublicationDetails`
--
  | 2024-01-10 20:21:13 UTC | --> src/sql-parser/src/ast/defs/ddl.rs:915:11
  | 2024-01-10 20:21:13 UTC | \|
  | 2024-01-10 20:21:13 UTC | 915 \|     /// [`mz_storage_types::sources::postgres::PostgresSourcePublicationDetails`]
  | 2024-01-10 20:21:13 UTC | \|           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no item named `mz_storage_types` in scope
  | 2024-01-10 20:21:13 UTC | \|
```

I'm a little hesitant to add `mz-storage-types` as a dependency of `mz-sql-parser` to satisfy this -- how do you feel about me removing the doc link format here (and for the mysql one below)?

Reply:
Sounds fine, remove the formatting :+1: